openshift/release/artifacts/serving-post-install-jobs.yaml

```diff
@@ -34,7 +34,7 @@ spec:
         app: storage-version-migration-serving
         app.kubernetes.io/name: knative-serving
         app.kubernetes.io/component: storage-version-migration-job
-        app.kubernetes.io/version: "v1.9.0"
+        app.kubernetes.io/version: "release-v1.9"
```

Comment:
Was this fixed manually?

Reply:
It was generated by `openshift/release/generate-release.sh "BRANCH_NAME"` so `release-v1.9` is the branch name.